frontend/packages/component-library/src/carousel/carousel.module.scss

```diff
@@ -1,5 +1,5 @@
-@import '@code-dot-org/component-library-styles/colors.scss';
-@import '@code-dot-org/component-library-styles/mixins.scss';
+@use '@code-dot-org/component-library-styles/colors';
```

Comment:
Not sure if we actually need this? Color variables still work without it in Storybook and Contentful.

Reply:
Got it, thanks for the explanation! (And I'll start using `@use` 🙌)